src/locales/hu.yaml

```diff
@@ -132,184 +181,319 @@ app:
+      accel_to_decel: Gyorsulás & Lassulás
       acceleration: Gyorsulás
+      actual_time: Tényleges
       add_camera: Kamera hozzáadása
-      add_preset: Előre beállított érték hozzáadása
-      api_url: API URL címe
-      confirm: Megerősítés
+      add_filter: Szűrő hozzáadása
+      add_preset: Előbeállítás hozzáadása
+      add_user: Felhasználó hozzáadása
+      api_key: Api kulcs
+      api_url: API URL
+      category: Kategória
+      change_password: Jelszó módosítása
+      clear_all: Mindent kitöröl
+      color: Szín
+      confirm: Megerősít
+      current_password: Jelenlegi jelszó
+      disabled_while_printing: Nyomtatás közben letiltva
       edit_camera: Kamera szerkesztése
-      edit_preset: Előre beállított érték szerkesztése
+      edit_filter: Szűrő szerkesztése
+      edit_preset: Előbeállítás szerkesztése
+      edit_user: Felhasználó szerkesztése
       extrude_length: Extrudálás hossza
       extrude_speed: Extrudálás sebessége
+      filament: Nyomtatószál
+      file: Fájl
+      finish_time: Befejezés
       flow: Átfolyás
       free: szabad
+      heaters_busy: A nyomtató jelenleg foglalt. A fűtőtestek kikapcsolása sikertelen nyomtatást eredményezhet.
       high: Magas
-      host: Gazda
+      host: Házigazda
+      layer: Réteg
       layout: Elrendezés
-      longest_job: Leghosszab meló
+      longest_job: A leghosszabb munka
       low: Alacsony
+      m117: M117
       name: Név
-      'off': Ki
-      power: Táp
+      new_password: Új jelszó
+      no_notifications: Nincsenek értesítések
+      'on': 'Be'
+      'off': 'Ki'
+      password: Jelszó
+      power: Áramellátás
       printers: Nyomtatók
-      requested_speed: Kért sebesség
+      progress: Folyamat
+      requested_speed: Sebesség
+      retract_length: Visszahúzás hossza
+      retract_speed: Visszahúzási sebesség
       save_as: Mentés másként
       services: Szolgáltatások
+      slicer: Szeletelő
       speed: Sebesség
-      sqv: Sebesség a Négyzet sarkainál
-      total_filament: A felhasznált összes filament
-      total_filament_avg: Átlag nyomtatásonként
+      sqv: Négyzet saroksebesség
+      total: Teljes
+      total_filament: Összes felhasznált nyomtatószál
+      total_filament_avg: Átl. nyomtatásonként
       total_jobs: Összes nyomtatási feladat
       total_print_time: Teljes nyomtatási idő
-      total_print_time_avg: Átlag nyomtatásonként
+      total_print_time_avg: Átl. nyomtatásonként
       total_time: Teljes idő
-      total_time_avg: Átlag nyomtatásonként
+      total_time_avg: Átl. nyomtatásonként
+      uncategorized: Nem kategorizált
+      unretract_extra_length: Extra hossz visszahúzása
+      unretract_speed: Visszahúzási sebesség
       used: használt
-      variance: Variálás
+      variance: Eltérés
       velocity: Sebesség
-      z_offset: Z eltolás
+      visible: Látható
+      z_offset: Z Eltolás
+      unsaved_changes: Nem mentett módosítások
+      alias: Álnév
```

Comment:
```suggestion
      alias: Becenév
```

Reply:
Well, I remember writing that there were still bugs in it. I am constantly working to improve them.

Thank you for your understanding!

By: AntoszHUN